src/praveen/vehicleDirection.test.ts

```diff
@@ -0,0 +1,50 @@
+import {
+  getFinalDirectionForVehicle,
+  getNextDirectionForMove
+} from './vehicleDirection';
+
+const testData = [
+  'STRAIGHT',
+  'RIGHT',
+  'LEFT',
+  'BACK',
+  'RIGHT',
+  'STRAIGHT',
+  'LEFT'
+];
+
+describe('testing getFinalDirectionForVehicle function', () => {
```

Comment:
Would the following test case look good???
```
describe('getFinalDirectionForVehicle function', () => {
  test('whether the function returns SOUTH direction when testData is provided  ', () => {
    const result = getFinalDirectionForVehicle(testData);
    expect(result).toEqual('SOUTH');
  });
```

Reply:
fixed